support/faq/are-you-sending-emails-as-encrypted-messages.md

```diff
@@ -1,6 +1,8 @@
 ---
 title: "Are you sending emails as Encrypted messages?"
-description: "Spark Post supports Transport Layer Security TLS TLS and its predecessor Secure Sockets Layer SSL are cryptographic protocols that provide security on the Internet and let you encrypt emails All incoming API calls use HTTPS and are SSL TLS encrypted Incoming SMTP offers TLS if the user choses to enable..."
+description: "SparkPost supports Transport Layer Security(TLS). TLS is a cryptographic protocol that provide security on the Internet and lets you encrypt emails during transit. All incoming API calls to SparkPost use HTTPS and are SSL/TLS encrypted. SparkPost also offers TLS support for incoming SMTP traffic if the user chooses to enable it."
 ---
 
-SparkPost supports Transport Layer Security (TLS). TLS and its predecessor, Secure Sockets Layer (SSL), are cryptographic protocols that provide security on the Internet and let you encrypt emails.  All incoming API calls use HTTPS and are SSL/TLS encrypted. Incoming SMTP offers TLS, if the user choses to enable *Encryption*: STARTTLS.  We also do opportunistic TLS for outbound traffic, which will depend on whether the receiving SMTP servers supports TLS.
+SparkPost supports the use of Transport Layer Security (TLS). TLS and its predecessor, Secure Sockets Layer (SSL), are cryptographic protocols that provide security on the Internet and let you encrypt emails while they are in transit. All incoming API calls to SparkPost use HTTPS and are SSL/TLS encrypted. SparkPost also offers TLS for incoming SMTP traffic if the user chooses to enable it through the use of "STARTTLS" SMTP extension. SparkPost strongly recommends using TLS with SMTP to protect your message content, recipient information and API keys in transmission. This includes API keys and any details such as recipient email addresses and message content. See more details about using encryption with SMTP <a href="https://developers.sparkpost.com/api/smtp/#header-smtp-security">here</a>.
```

Comment:
use of "STARTTLS" => use of the "STARTTLS"

Reply:
> This includes API keys and any details such as recipient email addresses and message content.

That sentence seems redundant given the previous sentence.

Reply:
Fixed both.